### DISCUSSION TITLE
Does strongswan support IPsec vpn aggregation

### DISCUSSION THREAD

**Discussion by liangfeng02** [strongswan/strongswan#1686] [General]
Hi, we current configure IPsec vpn aggregation between strongswan and foritgate.  we had two Internets on foritgate,  and one for Strondswan.   I create two IPSEC01, and ipsec02, and IPsec agg,  add both ipsec 01 and ipsec 02 into ipsecagg.  when I enable both lines , there are some issues,  lost package, not ping from one side or the other side. I ping from fortigate side to Strongswan.  I tried to debug on fortigate,  it seems one of tunnel ipsec02 can sent icmp, but can't get response from strongswan,   but at the same time my pc on the sslvpn is working on another tunnel ipsec01.   here is the config of strongswan

conn %default
authby=secret
auto=start
type=tunnel
left=10.151.2.133
leftid=10.151.2.133
leftsubnet=10.151.2.128/26,10.151.51.0/24
rightsubnet=10.245.220.0/22
keyexchange=ikev2
dpdaction=clear
dpddelay=300s
rekey=no
ike=aes256-sha1-modp1536!
esp=aes256-sha1-modp1536!
lifttime=8h

conn leihuo-mtl-1
right=public IP 1

conn leihuo-mtl-2
right=public ip 2

**Comment by tobiasbrunner** [comment]
Difficult to say. I don't know exactly what FortiGate's IPsec aggregation feature entails, maybe it requires using a [route-based approach](https://docs.strongswan.org/docs/5.9/features/routeBasedVpn.html) and running a routing protocol like OSPF over the whole thing.

Regarding the immediate issues you are seeing, you'll have to look at the logs and statistics (in the status output) to see what exactly is going on and which SAs are being used and are working.
